src/vuex/modules/data/actions.js

```diff
@@ -28,8 +30,22 @@ export const listIndexesAndCollections = (store) => {
               reject(new Error(error.message))
               return
             }
-
             if (index !== '%kuzzle') {
+              // realtime collections
+              // eslint-disable-next-line no-undef
+              let realtimeCollections = JSON.parse(localStorage.getItem('realtimeCollections') || '[]')
+              // Cannot use realtimeCollections.find(...) as it is a prototype in ES6 and is not transpiled by babel
+              for (var i = 0; i < realtimeCollections.length; i++) {
+                if (realtimeCollections[i].index === index) {
+                  currentIndex = realtimeCollections[i]
+                  break
+                }
+              }
+              if (currentIndex) {
+                result.realtime = realtimeCollections.map(o => {
```

Comment:
Why don't directly use `realtimeCollections.map`?
Ok, we will loop on "all" realtimeCollections even if there is no realtime collection for this index... but... I think it's clearer and about perf... Array.prototype.map = ~10000000 ops/sec :)